test/jasmine/tests/pie_test.js

```diff
@@ -1609,14 +1615,15 @@ describe('Test event data of interactions on a pie plot:', function() {
             Plotly.restyle(gd, {
                 labels: [labels.concat(labels)],
                 values: [values.concat(values)]
-            });
-
-            click(pointPos[0], pointPos[1]);
-            expect(futureData.points.length).toEqual(1);
+            })
+            .then(function() {
+                click(pointPos[0], pointPos[1]);
+                expect(futureData.points.length).toEqual(1);
 
-            expect(futureData.points[0].pointNumber).toBeUndefined();
-            expect(futureData.points[0].i).toBeUndefined();
-            expect(futureData.points[0].pointNumbers).toEqual([4, 9]);
+                expect(futureData.points[0].pointNumber).toBeUndefined();
+                expect(futureData.points[0].i).toBeUndefined();
+                expect(futureData.points[0].pointNumbers).toEqual([4, 9]);
+            });
```

Comment:
`done`/`failTest`

Reply:
Fixed in https://github.com/plotly/plotly.js/pull/5340/commits/33b72e1eac53fb93c7b6d71045210e8847312d04